### PR TITLE
Localization #10

### LINKED ISSUE #10: Front localization
Необходимо добавить поддержку локализации с помощью i18n библиотеки. 

В рамках задачи необходимо 

- добавить библиотеку в зависимости, 
- развернуть инфрастраструктуру, словари для английского и украинского языка
- добавить возможность выбора языка приложения
- сделать переводы для существующих компонентов.

Ссылка на материалы по локализации в VueJS [здесь](https://kazupon.github.io/vue-i18n/introduction.html)

Ориентировочное время выполнения (Estimated time): 12ч

### PR DESCRIPTION
**Pull request by Yaronberg**
I did it. Left to make layout

### CODE REVIEW

File: front/src/components/index.js [hunk 1]
```diff
@@ -1 +1,2 @@
 export { default as FfLogo } from './logo/logo.component.vue';
+export { default as FfLocale } from './ff-locale-select/ff-locale-select.component.vue';
```
Comment (horba) [inline front/src/components/index.js]: FfLocaleSelect

File: front/src/layouts/menu/menu.layout.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
-import { FfLogo } from '@components';
+import { FfLocale, FfLogo } from '@components';
```
Comment (horba) [inline front/src/layouts/menu/menu.layout.js]: FfLocaleSelect

File: front/src/layouts/menu/menu.layout.html [hunk 3]
```diff
@@ -15,4 +15,7 @@
       <el-button icon="el-icon-search" class="second-color"></el-button>
       <el-button class="second-color">ВОЙТИ</el-button> 
     </div> 
+    <div>
+      <ff-locale />
```
Comment (horba) [inline front/src/layouts/menu/menu.layout.html]: ff-locale-select

File: front/src/store/modules/locale.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        state.language = language;
+      }
+    },
+    CHANGE_LANGUAGE (state, language) {
+      i18n.locale = language;
+      localStorage.setItem('language', language);
+      state.language = language;
+    }
+  }
+};
```
Comment (horba) [inline front/src/store/modules/locale.js]: locale.store.js

Comment (horba) [reply front/src/store/modules/locale.js]: https://github.com/horba/film-finity/wiki/Frontend-Namings-and-Structure

File: front/src/router/index.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
+import { Cartoons, FfCelebrities, Movies, News, Serials } from '@views';
+
```
Comment (horba) [inline front/src/router/index.js:2]: why do you change it?

File: front/src/components/ff-locale-select/ff-locale-select.component.html [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<div class="locale-select">
```
Comment (horba) [inline front/src/components/ff-locale-select/ff-locale-select.component.html]: ff-locale-select

File: front/src/components/ff-locale-select/ff-locale-select.component.html [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+  <el-select v-model="value" placeholder="Select">
+    <el-option
+      v-for="item in locales"
```
Comment (horba) [inline front/src/components/ff-locale-select/ff-locale-select.component.html]: locale in locales

File: front/src/locales/index.js [hunk 8]
```diff
@@ -0,0 +1,9 @@
+import en from './translations/en';
+import ru from './translations/ru';
+import ua from './translations/ua';
+
+export default {
+  en,
+  ru,
+  ua
+};
```
Comment (horba) [inline front/src/locales/index.js:9]: add path `@locales` to vue config

File: front/src/plugins/i18n.js [hunk 9]
```diff
@@ -0,0 +1,14 @@
+import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+
+Vue.use(VueI18n);
+
+export default new VueI18n({
+  locale: window.localStorage.Language || 'ru',
```
Comment (horba) [inline front/src/plugins/i18n.js]: Language or language?

**Review by horba — COMMENTED** [review]
seems, you have some mess with distribution of locale management between store and app.vue

**Comment by horba** [comment]
it is just manual work to add translation into existing templates via $t function/ please, make it for at least for menu and footer layouts and then we can finish task

File: front/src/router/index.js (hunk 5 above)
Comment (Yaronberg) [reply front/src/router/index.js:2]: i dont change it, linter or formater make this.

**Review by horba — APPROVED** [review]

**Pull request merged**